timer/tablestore/sql.go

```diff
@@ -104,7 +109,7 @@ func buildInsertTimerSQL(dbName, tableName string, record *api.TimerRecord) (str
 		"EVENT_DATA, "+
 		"SUMMARY_DATA, "+
 		"VERSION) "+
-		"VALUES (%%?, %%?, %%?, 'TIDB', %%?, %%?, %%?, %s, %%?, %%?, %%?, %%?, %s, %%?, %%?, 1)",
+		"VALUES (%%?, %%?, %%?, 'TIDB', %%?, %%?, %%?, %s, %%?, JSON_MERGE_PATCH('{}', %%?), %%?, %%?, %s, %%?, %%?, 1)",
```

Comment:
Why is it a `JSON_MERGE_PATCH` but not `cast(? as JSON)`?

Reply:
That is because `timerExt` object will be used in both insert and update. If we simply cast it to json. We will write some null value to it. For example, when update, the "manual" field should be serialized to `{"request_id": "123", "event_id": null, ...}` to make a full update and to make sure "event_id" is removed. And in insert scene, we want to keep the same behavior